docs/tutorials/hpa-recommendation.zh.md

```diff
@@ -0,0 +1,83 @@
+# 弹性推荐
+
+通过弹性推荐，你可以发现集群中适合弹性的资源，并使用 Crane 推荐的弹性配置创建自动弹性器: EffectiveHPA
+
+## 创建弹性分析
+
+创建一个**弹性分析** `Analytics`，这里我们通过 deployment: `craned` and `metric-adapter` 作为一个例子
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/gocrane/crane/main/examples/analytics/analytics-hpa.yaml
+kubectl get analytics -n crane-system 
+```
+
```

Comment:
![2022-05-12 22 29 54](https://user-images.githubusercontent.com/35299017/168099486-55f95c36-31da-4309-a806-8765269348c3.gif)

We can use this feature to show details if the user wants.

**Don't forget to enable [this feature](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#details) in `mkdocs.yml`.**
```yaml
markdown_extensions:
  - pymdownx.details
```

---

```suggestion

??? example "analytics-hpa.yaml"

      ```yaml
      apiVersion: analysis.crane.io/v1alpha1
      kind: Analytics
      metadata:
        name: craned-hpa
        namespace: crane-system
      spec:
        type: HPA                        # This can only be "Resource" or "HPA".
        completionStrategy:
          completionStrategyType: Periodical  # This can only be "Once" or "Periodical".
          periodSeconds: 600                  # analytics selected resources every 10 minutes
        resourceSelectors:                    # defines all the resources to be select with
          - kind: Deployment
            apiVersion: apps/v1
            name: craned

      ---

      apiVersion: analysis.crane.io/v1alpha1
      kind: Analytics
      metadata:
        name: metric-adapter-hpa
        namespace: crane-system
      spec:
        type: HPA                       # This can only be "Resource" or "HPA".
        completionStrategy:
          completionStrategyType: Periodical # This can only be "Once" or "Periodical".
          periodSeconds: 3600                # analytics selected resources every 1 hour
        resourceSelectors:                   # defines all the resources to be select with
          - kind: Deployment
            apiVersion: apps/v1
            name: metric-adapter
      ```

```